### DISCUSSION TITLE
Seeing and Commenting on commits.

### DISCUSSION THREAD

**Discussion by NeeEoo** [community/community#3812] [Mobile]
It would be good to be able to comment on a line or commit. And being able to read comments on commits.

**Comment by eliperkins** [comment]
Thanks for the feedback, @NeeEoo !

> It would be good to be able to comment on a line or commit

Per-line comments on pull requests are supported in GitHub mobile today. Tapping on a line of code in a pull request from the Files Changed view will allow you to live a comment on a specific line of code.

Commit comments and per-line comments on commits are on our radar, but we don't have a timeline for this feature currently. I've added your notes to our internal tracking issue!

**Reply by KalleOlaviNiemitalo** [reply]
In the app, I got a notification about a comment on a commit:

![Notification about commit comment in GitHub mobile app](https://user-images.githubusercontent.com/46201428/133729081-979fa41f-ad9e-4ff5-9f67-71f55eed2ad1.png)

I clicked the notification but did not see any comment:

![Changes tab of commit in GitHub mobile app](https://user-images.githubusercontent.com/46201428/133729175-40058b3d-f5a7-4c77-acda-ccd3f503765b.png)

![Details tab of commit in GitHub mobile app](https://user-images.githubusercontent.com/46201428/133729288-2ab6f435-5c06-4810-a9cd-ed229193f2ef.png)

I then chose Share, and Open in Safari. [That](https://github.com/dotnet/sdk/commit/d239c27aac0fc61380ad8f408ad757de286467c3) displayed the comment all right:

![image](https://user-images.githubusercontent.com/46201428/133729439-0265f80f-59ae-41ca-870c-62b0ba7a4ab6.png)

This makes a confusing user experience. Until the app supports comments on commits, it might be better to make it open Safari directly when I click such a notification.

**Comment by SPRZB** [comment]
Any update on this?

**Reply by Gitnitdun** [reply]
Hacked

**Comment by VuiMuich** [comment]
Adding to the little annoyance is, that following the notification and not seeing the comment still marks it as `read`, whereas in the browser it regularly remains unread, even after scrolling through the whole conversation.

**Reply by Gitnitdun** [reply]
Quit

**Comment by Gitnitdun** [comment]
Stop

**Comment by Gitnitdun** [comment]
Hacking me

**Reply by Gitnitdun** [reply]
Yes I am being followed by whoever is all involved in this organization is octo-org, octocat,Alvarado group, contractors

**Comment by Pjrich1313** [comment]
I know I have had someone in my account trying to erase my codes I want them replaced and all my copilot put back
